docs/send-data/index.md

```diff
@@ -7,77 +7,61 @@ tags: [send data, collectors, collection]
   </div>
 </div>
-  <div className="box smallbox3 card">
+  <div className="box smallbox4 card">
   <div className="container">
-  <img src={useBaseUrl('img/icons/operations/collect.png')} alt="icon" width="40"/><h4>Other Data Sources</h4>
-  <ul>
-  <li><a href="/docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs">Amazon Cloudwatch</a></li>
-  <li><a href="/docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf">Telegraf</a></li>
-  <li><a href="/docs/send-data/collect-from-other-data-sources">Azure, Heroku, Docker, K8s, and more</a></li>
-  </ul>
+  <a href="/docs/send-data/collect-from-other-data-sources"><img src={useBaseUrl('img/icons/operations/collect.png')} alt="icon" width="40"/><h4>Other Data Sources</h4></a>
+  <p>Collect logs and metrics from a variety of other data sources.</p>
   </div>
 </div>
-<div className="box smallbox4 card">
+<div className="box smallbox5 card">
   <div className="container">
-  <img src={useBaseUrl('img/icons/operations/collect.png')} alt="icon" width="40"/><h4>JSON Configuration</h4>
-  <ul>
-  <li><a href="/docs/send-data/use-json-configure-sources/json-parameters-hosted-sources">Hosted Collectors</a></li>
-  <li><a href="/docs/send-data/use-json-configure-sources/json-parameters-installed-sources">Installed Collectors</a></li>
-  <li><a href="/docs/send-data/use-json-configure-sources/local-configuration-file-management">Local Config File Mgmt</a></li>
-  </ul>
+  <a href="/docs/send-data/use-json-configure-sources/json-parameters-hosted-sources"><img src={useBaseUrl('img/icons/operations/collect.png')} alt="icon" width="40"/><h4>JSON Configuration</h4></a>
+  <p>Learn how to configure Installed Collector and Hosted Collector sources using JSON files.</p>
   </div>
 </div>
-  <div className="box smallbox5 card">
+  <div className="box smallbox6 card">
     <div className="container">
-    <img src={useBaseUrl('img/icons/operations/collect.png')} alt="icon" width="40"/><h4>Source Reference Information</h4>
-    <ul>
-    <li><a href="/docs/send-data/reference-information/time-reference">Timestamps, Time Zones / Ranges, Date Formats</a></li>
-    <li><a href="/docs/send-data/reference-information/use-wildcards-paths">Wildcards in Paths</a></li>
-    </ul>
+    <a href="/docs/send-data/reference-information/time-reference"><img src={useBaseUrl('img/icons/operations/collect.png')} alt="icon" width="40"/><h4>Timestamps, Time Zones Reference</h4></a>
+    <p>Learn about supported timestamps, date formatting, and more. </p>
     </div>
   </div>
-  <div className="box smallbox6 card">
+  <div className="box smallbox7 card">
+    <div className="container">
+    <a href="/docs/send-data/reference-information/use-wildcards-paths"><img src={useBaseUrl('img/icons/operations/collect.png')} alt="icon" width="40"/><h4>Wildcards in Paths</h4></a>
+    <p>Use wildcards in Source paths to collect all files of a certain type.</p>
```

Comment:
Why do we have this section in the Send Data Page ? Shouldn't this be within the Installed Collector, Otel Collector Pages, since it relates how we should set things up?

Reply:
So this does not apply to Hosted Collectors? Installed and OTel only?

Reply:
In the current structure, it looks like all reference material is here: https://help.sumologic.com/docs/send-data/reference-information. We can definitely look into this after S3M.